section1/set8/nestedLoops/challenge.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+func main() {
+	n, err := strconv.Atoi(os.Args[1])
+
+	if err != nil {
+		fmt.Println("Provide an integer.")
+		return
+	}
+
+	fmt.Printf("%5s", "X")
```

Comment:
include comments like what is the significance of these prints, for e.g. say something like printing n nums for debugging etc

Reply:
fixed

Reply:
It looks good now. Thanks